### ISSUE TITLE
Crop Avatar and banner before posting

### ISSUE THREAD

**Issue by trymeouteh** [LemmyNet/lemmy-ui#364]
Please add the tools to be able to crop your avatar and banner picture before you upload it to your profile.

**Comment by dessalines** [comment]
I don't think we want to bundle a whole image editing suite into lemmy. But if someone finds a very simple crop library,  I'll consider it.

**Comment by trymeouteh** [comment]
CropperJS?

https://github.com/oscarkey/cropper.js

**Comment by dessalines** [comment]
Very low priority for me, but if someone does a PR, I'd take a look.

**Comment by theo-mitchell** [comment]
I can take a look if this is still open. Please let me know if I should go ahead.

**Comment by CTalvio** [comment]
I would want this.

**Comment by theo-mitchell** [comment]
@CTalvio sure thing, Ill take a stab then.
